templates/etc/ansible/facts.d/nginx.fact.j2

```diff
@@ -21,4 +21,5 @@
 {% endif %}
 "user": "{{ nginx_user }}",
 "www": "{{ nginx_root_www_path }}"
```

Comment:
You either need to add a comma here or move `flavor` 1 line up, otherwise local facts will not be parsed correctly by Ansible.